sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactDataForm.java

```diff
@@ -960,4 +981,19 @@ private void initContactDateValidation(DateField startDate, DateField endDate, C
 			}
 		});
 	}
+
+	@Override
+	public void setValue(ContactDto newFieldValue) throws ReadOnlyException, Converter.ConversionException {
+		super.setValue(newFieldValue);
+
+		expectedUntilDate.setValue(
```

Comment:
this returns nullPointerException. If the Case follow-up feature is not activated. You are trying to set a value on a field that does not exist. Please update.